modules/extensions-support/src/test/java/org/mule/runtime/module/extension/internal/loader/DefaultExtensionModelFactoryTestCase.java

```diff
@@ -87,17 +88,20 @@ public void flyweight() {
 
   @Test
   public void blockingExecutionTypes() {
-    ExtensionModel extensionModel = createExtension(HeisenbergExtension.class);
+    final List<String> blockingOperations = Arrays.asList("killMany", "executeAnything", "alwaysFailsWrapper",
```

Comment:
why are scopes marked as blocking? Can't we have NB chains?

Reply:
bad var name, these are nonBlockingOperations, check the assertion:
`assertThat(operation.isBlocking(), is(!nonBlockingOperations.contains(operationName)));`